content/docs/dvclive/api-reference/live/log_params.md

```diff
@@ -0,0 +1,54 @@
+# Live.log_params()
+
+Logs the given group of parameters `params`, saving them as yaml.
+
+```py
+ def log_params(params: Dict[ParamLike]):
+```
+
+Also see `Live.log_param()`.
+
+## Usage
+
+```py
+from dvclive import Live
+
+live = Live()
+
+params = {
+    "lr": 1e-3,
+    "n_iter": 50
+}
+live.log_params(params)
+```
+
+## Description
+
+On each `live.log_params(params)` call, DVCLive will write keys/values pairs in
+the `params` dict to `{Live.params_file}`:
```

Comment:
```suggestion
the `params` dict to `{Live.params_path}`:
```

Also, if this becomes `{Live.params_path}.yaml`, I think that might be enough to indicate the output format.

Reply:
> Should it be {Live.params_path}.yaml to match summary_path (sorry this wasn't raised in the core PR)?
...
> Also, if this becomes {Live.params_path}.yaml, I think that might be enough to indicate the output format.

I'd like to have something like that, but I'd expect `dvclive.params_path` to be a full path (e.g. `dvclive/params.yaml`) Having something like `dvclive/params` would be pretty odd.